gensrc/script/gen_functions.py

```diff
@@ -151,6 +151,8 @@ def generate_fe(path):
         'functionSet.addVectorizedScalarBuiltin(${id}, "${name}", ${has_vargs}, Type.${ret}${args_types});')
 
     def gen_fe_fn(fnm):
+        if fnm['id'] in functions.fe_deprecated_functions:
```

Comment:
why do we need functions.fe_deprecated_functions? why could we work before without functions.fe_deprecated_functions?